src/init.cpp

```diff
@@ -1619,19 +1619,19 @@ bool AppInitMain()
         return false;
     }
 #endif
-    if (g_logger->m_print_to_file) {
-        if (gArgs.GetBoolArg("-shrinkdebugfile", g_logger->DefaultShrinkDebugFile())) {
+    if (LogInstance().m_print_to_file) {
+        if (gArgs.GetBoolArg("-shrinkdebugfile", LogInstance().DefaultShrinkDebugFile())) {
             // Do this first since it both loads a bunch of debug.log into memory,
             // and because this needs to happen before any other debug.log printing
-            g_logger->ShrinkDebugFile();
+            LogInstance().ShrinkDebugFile();
         }
     }
-    if (!g_logger->StartLogging()) {
+    if (!LogInstance().StartLogging()) {
         return InitError(strprintf("Could not open debug log file %s",
-                                   g_logger->m_file_path.string()));
+                                   LogInstance().m_file_path.string()));
```

Comment:
Missing the indentation changes upstream did https://github.com/bitcoin/bitcoin/pull/15266/files#diff-b1e19192258d83199d8adaa5ac31f067af98f63554bfdd679bd8e8073815e69dL1208-R1208

Reply:
Fixed Indentation

Reply:
not sure why after this indentation change 
rpc_setban.py test is failing 

@PastaPastaPasta @UdjinM6